bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-simditor-mention",
-  "version": "4.0.0-alpha.1",
+  "version": "4.0.1",
```

Comment:
We should bump it to `4.0.0-alpha.2` instead.

The idea is to work towards a stable `4.0.0` release by incrementally improving what this library does. That `4.0.0` release will be our gift to the community. Until then, every version is alpha.